scripts/extract-translations.js

```diff
@@ -65,70 +71,72 @@ const new_find_keys = (file) => {
         keys.push(pure_text.replace(/\\/g, ''))
         result = new_i18n_marker.exec(file)
     }
-    return keys;
+    return keys
 }
 
 /** **********************************************
  * Compile
  */
-extractTranslations();
+extractTranslations()
 
 function getStrapiStrings(strapi_obj_data) {
-    return strapi_obj_data instanceof Array ? strapi_obj_data.map((array) => getStrapiStrings(array)).flat() :
-        strapi_obj_data instanceof Object ? Object.values(strapi_obj_data).map((object) => getStrapiStrings(object)).flat() :
-            strapi_obj_data;
+    return strapi_obj_data instanceof Array
+        ? strapi_obj_data.map((array) => getStrapiStrings(array)).flat()
+        : strapi_obj_data instanceof Object
+        ? Object.values(strapi_obj_data)
+              .map((object) => getStrapiStrings(object))
+              .flat()
+        : strapi_obj_data
 }
 
 function extractTranslations() {
-    (async () => {
+    ;(async () => {
```

Comment:
```suggestion
    (async () => {
```

Reply:
hi @Nuzhy-Deriv this changes are due to prettier 
can you please check the second commit it will be more clear